### ISSUE TITLE
Game crash on creating new world

### ISSUE THREAD

**Issue by Solariser** [anegostudios/VintageStory-Issues#702]
To replicate: Click create new world, survival mode.

Specs:
- Windows 10
- 16GB Ram
- Intel core i7-8700 CPU
- Nvidia Geforce GTX 1660

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.14.2 (Stable)2020-12-18 16:01:40: Critical error occurred
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.ThrowHelper.ThrowKeyNotFoundException()
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Vintagestory.GameContent.EntityBehaviorExtraSkinnable.Essr_OnTesselation(Shape& entityShape, String shapePathForLogging)
   at Vintagestory.GameContent.EntityShapeRenderer.TesselateShape()
   at Vintagestory.GameContent.EntitySkinnableShapeRenderer.TesselateShape()
   at Vintagestory.GameContent.GuiDialogCreateCharacter.ComposeGuis()
   at Vintagestory.GameContent.GuiDialogCreateCharacter.OnGuiOpened()
   at Vintagestory.API.Client.GuiDialog.TryOpen()
   at Vintagestory.GameContent.CharacterSystem.Event_PlayerJoin(IClientPlayer byPlayer)
   at Vintagestory.Client.NoObf.GeneralPacketHandler.HandlePlayerData(_yYcLwwVgddc6OWOmqOfREe6xnKd packet)
   at Vintagestory.Client.NoObf.ClientMain.ExecuteMainThreadTasks(Single deltaTime)
   at _dUwwv9Q3M6hPimkiHEIazbxLr9g._X2niSFpb8ndF6zWsCcobp5QBcvO(Single )
   at _ydpin5JFjepNgd3JOZRb2z5I5Fg._LvRZFcsDWyPwKLB1F2UK7fQ2N6d(Single )
   at _ydpin5JFjepNgd3JOZRb2z5I5Fg._GO2qaUrk9dl0IceTLiA7P4LcpAB(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _Ine8FeGTtDXciSIfaZMHMwCLKGBA._8puLW8J44fiq3A5hthQTfB8WNxc(_SbVlAI6dJIZ4JRt0da0lu23DF14 , String[] )
   at _lErb0L2e9E9htuT2VVnAsXIZqc7A._8puLW8J44fiq3A5hthQTfB8WNxc(ThreadStart )
-------------------------------
```

```
Event Log, last 3 entries
==================================
{ TimeGenerated = 2020-12-18 08:10:42, Site = , Source = .NET Runtime, Message = Application: Vintagestory.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0020001, exception address 00007FF8DE33D759
Stack:
   at OpenTK.Graphics.OpenGL.GL.FramebufferTexture2D(OpenTK.Graphics.OpenGL.FramebufferTarget, OpenTK.Graphics.OpenGL.FramebufferAttachment, OpenTK.Graphics.OpenGL.TextureTarget, Int32, Int32)
   at Vintagestory.Client.NoObf.ClientPlatformWindows.SetupDefaultFrameBuffers()
   at Vintagestory.Client.NoObf.ClientPlatformWindows.RebuildFrameBuffers()
   at Vintagestory.Client.NoObf.ClientPlatformWindows.Window_Resize(System.Object, System.EventArgs)
   at System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   at OpenTK.GameWindow.OnResize(System.EventArgs)
   at System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   at OpenTK.Platform.Windows.WinGLNative.HandleWindowPositionChanged(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   at OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   at OpenTK.Platform.Windows.Functions.PeekMessage(OpenTK.Platform.Windows.MSG ByRef, IntPtr, Int32, Int32, OpenTK.Platform.Windows.PeekMessageFlags)
   at OpenTK.Platform.Windows.Functions.PeekMessage(OpenTK.Platform.Windows.MSG ByRef, IntPtr, Int32, Int32, OpenTK.Platform.Windows.PeekMessageFlags)
   at OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   at OpenTK.GameWindow.Run(Double, Double)
   at _Ine8FeGTtDXciSIfaZMHMwCLKGBA._8puLW8J44fiq3A5hthQTfB8WNxc(_SbVlAI6dJIZ4JRt0da0lu23DF14, System.String[])
   at _lErb0L2e9E9htuT2VVnAsXIZqc7A._8puLW8J44fiq3A5hthQTfB8WNxc(System.Threading.ThreadStart)
   at _Ine8FeGTtDXciSIfaZMHMwCLKGBA._Q94MSQMy7MbChZ1FRrVUB2AgwfL(System.String[])
 }
```

**Comment by copygirl** [comment]
Does this happen every time you create a new world, only sometimes, or has it only happened once?
Do you use any mods?

**Comment by Solariser** [comment]
> 
> 
> Does this happen every time you create a new world, only sometimes, or has it only happened once?
> Do you use any mods?

No mods, only happened one time so far.

**Comment by copygirl** [comment]
Could you attach your full `client-main.txt` log file?

**Comment by Solariser** [comment]
> 
> 
> Could you attach your full `client-main.txt` log file?

Yeah sure, I have been playing since this issue occurred so I'm not sure if that affects the file.

[client-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/5716864/client-main.txt)

**Comment by Solariser** [comment]
> 
> 
> Could you attach your full `client-main.txt` log file?

This one might be more useful?

[client-crash.txt](https://github.com/anegostudios/VintageStory-Issues/files/5716872/client-crash.txt)

**Comment by copygirl** [comment]
Ah yes, that may have been a bit silly of me. The log file would only have helped me if you had not restarted the game since the crash, which you probably did. I hope the dev team will have the chance to look at this. In the meantime, I will recommend them saving some additional information when crashes do happen.

**Comment by tyronx** [comment]
I made some fixes today related to this, there is a chance this is fixed in v1.14.3.

**Comment by copygirl** [comment]
Will assume this is fixed. Please respond if not.
